jbpm-designer-backend/src/main/java/org/jbpm/designer/web/server/SimulationServlet.java

```diff
@@ -526,6 +526,11 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
                 resp.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
                 PrintWriter pw = resp.getWriter();
                 pw.write(e.getMessage());
+			} catch (Throwable t) {
```

Comment:
I know this was already merged, but still, should we really catch `Throwable` here? This will catch all the errors (like OOME) as well. That's not usually something we want.